src/test/shell/bazel/BUILD

```diff
@@ -369,14 +369,14 @@ sh_test(
     ],
 )
 
-sh_test(
-    name = "bazel_rules_cc_test",
-    srcs = ["bazel_rules_cc_test.sh"],
-    data = [
-        ":test-deps",
-        "@bazel_tools//tools/bash/runfiles",
-    ],
-)
+# sh_test(
```

Comment:
Same comment about using "noci", and whether this test can be re-enabled.

Reply:
nope, just need that update

Reply:
I'll just leave these as broken since they shouldn't ever land this way